pkg/audit/service.go

```diff
@@ -21,6 +21,7 @@ import (
 // Config contains configurable values for audit service
 type Config struct {
 	MaxRetriesStatDB  int           `help:"max number of times to attempt updating a statdb batch" default:"3"`
+	StartDelay        time.Duration `help:"how long should we wait before starting the audit" default:"30s"`
```

Comment:
Is there a better way to delay things rather than using a hardcoded delay? e.g. waiting for some other event to finish?

Reply:
I agree with Egon, since if that event happens to take longer than 30s this won't work as expected. Or if that event takes a really short amount of time, 30s may be unnecessarily long.